lib/utils.js

```diff
@@ -327,6 +327,11 @@ const references = (types) => {
  */
 const fill = (types, type, refs, database) => {
   const ct = _.cloneDeep(type);
+  const placeholder = {
+    value: '',
+    label: '-- Choose One --',
```

Comment:
There should be an optional way to change this text. Can you please add a [setting option for references](https://github.com/punchcard-cms/punchcard/blob/master/input-plugins/input-plugin-reference/index.js#L32) with this text as default?